lunes_cms/cms/models/static.py

```diff
@@ -33,6 +33,15 @@ class Static:
         ("Adverb", "Adverb"),
     ]
 
+    # Possible grammatical genders
+    grammatical_genders = [
+        (0, "keiner"),
```

Comment:
I'm not sure if it says "keiner" in the specifications given by the Lunes team, but intuitively I would prefer 

```suggestion
        (0, "kein"),
```

for "kein Genus", see [here](https://www.duden.de/rechtschreibung/Genus)